src/test/kotlin/lotto/domain/LottoNumberTest.kt

```diff
@@ -9,8 +9,8 @@ import org.junit.jupiter.params.provider.CsvSource
 class LottoNumberTest {
     @ParameterizedTest
     @CsvSource("1, 1", "2, 2", "3, 3")
-    fun `from은 동일한 수를 내보낸다`(param: Int, expected: String) {
-        assertThat(LottoNumber.from(param).toString()).isEqualTo(expected)
+    fun `from은 동일한 수를 내보낸다`(param: Int, expected: Int) {
+        assertThat(LottoNumber.from(param)).isEqualTo(LottoNumber.from(expected))
```

Comment:
현재는 동일한 함수를 호출하기 때문에 테스트의 의미가 없어지겠네요ㅠㅠㅠ
* private constructor의 접근제한자를 수정
* value의 접근제한자를 수정(저는 요걸 권장드려요!)
하여 아래와 같이 수정하는 것도 좋아보여요!

```suggestion
        assertThat(LottoNumber.from(param)).isEqualTo(LottoNumber(expected))
```

하지만 테스트를 위해 접근제한자를 수정한다는게 맘에 드시지 않을 수 있으니...
한가지 방법을 안내드리자면 아래와 같습니다!! (리플렉션을 사용한 방법인데여! 웬만하면 위의 방법을 사용하시길 권장드립니다..!!)

```kotlin
    @ParameterizedTest
    @CsvSource("1, 1", "2, 2", "3, 3")
    fun `from은 동일한 수를 내보낸다`(param: Int, expected: Int) {
        val constructors = LottoNumber::class.java.declaredConstructors
        val constructor = constructors[0]
        constructor.isAccessible = true
        val expectedLottoNumber = constructor.newInstance(expected)
        //value class의 equals를 통해 객체를 비교함
        assertThat(LottoNumber.from(param)).isEqualTo(expectedLottoNumber)
    }
```